source/streaming-mode/streaming-read-config.txt

```diff
@@ -46,6 +46,27 @@ You can configure the following properties when reading data from MongoDB in str
    * - ``collection``
```

Comment:
The information from the "Escaping" section of "Documentation Changes Summary" is missing from the documentation of the `collection` configuration option. Those rules should be specified here in full.

P.S. This comment does not imply any changes to `release-notes.txt` in addition to what has already been done.

Reply:
Added to the configuration options page

Reply:
Let's not forget to specify this part:

> Note that if the value is specified as a string literal in Java code, then each reverse solidus has to be further escaped, leading to having to specify "\\\\".

[optional]
I think examples (like those given in "Documentation Changes Summary") are helpful and recommend (but don't insist, nor can I) including them.

Reply:
Updated to add examples and add the note for Java string literals

Reply:
✅ This is resolved, but I don't have the technical ability to resolve the discussion.